### PR TITLE
mass fraction runtime parameter cleaning

### PR DESCRIPTION
**Pull request by zingale**
Lots of tests had the same code to interpret mass fraction runtime parameters.  This consolidates them all.

### CODE REVIEW

**Comment by zingale** [comment]
tests pass:
http://groot.astro.sunysb.edu/Microphysics/test-suite/gfortran/2022-12-08-001/index.html

File: unit_test/test_sdc_vode_rhs/vode_rhs_test.H [hunk 1]
```diff
@@ -18,100 +18,8 @@ void do_vode_rhs()
     Real massfractions[NumSpec] = {-1.0};
 
     for (int n = 1; n <= NumSpec; ++n) {
-        switch (n) {
```
Comment (zhichen3) [inline unit_test/test_sdc_vode_rhs/vode_rhs_test.H:21]: need to include `react_util.H` ? Also, can't seem to even compile this test due to missing `vode_rhs_simplified_sdc.H`.

Comment (zingale) [reply unit_test/test_sdc_vode_rhs/vode_rhs_test.H:22]: should be fixed now

**Review by zhichen3 — APPROVED** [review]
running with `test_sdc_vode_rhs` gives an error saying `integration failed in net`, but I guess it's a separate issue.

**Pull request merged**